schema/inet.go

```diff
@@ -12,6 +13,12 @@ type InetTransformer interface {
 	TransformInet(*Inet) interface{}
 }
 
+// workaround this Golang bug: https://github.com/golang/go/issues/35727
+type inetWrapper struct {
```

Comment:
Maybe add a comment that this is only used in Unmarshal.
Maybe order structs so that exported struct `Inet` is at the top.